website/docs/cloud-docs/workspaces/settings/drift-detection.mdx

```diff
@@ -0,0 +1,56 @@
+---
+
+# Drift Detection
+
+-> **Note:** Drift detection is in beta and is available in the paid **Business** tier. Refer to [Terraform Cloud pricing](https://www.hashicorp.com/products/terraform/pricing) for details and  [provide feedback](https://forms.gle/BwjgvrKQcLbgqK3f7) about how we can improve this feature.
+
+Infrastructure drift means that your real-world infrastructure no longer matches the configuration in your Terraform state file. Drift occurs when a user modifies resources outside of the Terraform workflow. For example, a colleague may update resource configuration directly in the cloud provider console during a production incident, changing the resource attributes from those in tracked in your state file.
+
+> **Hands On:** Try our [Manage Resource Drift](https://learn.hashicorp.com/tutorials/terraform/resource-drift) tutorial on HashiCorp Learn to review how to detect and resolve drift.
+
+Drift detection is a workspace-level setting. When enabled, Terraform Cloud will automatically run a drift detection assessment for that workspace every 24 hours. The assessment checks the actual settings of your infrastructure against those tracked in your workspace's state file.
+
+## Requirements
+
+Drift detection requires the following settings on your workspace: 
+- Terraform version 0.15.4+
+- [Remote execution mode](/cloud-docs/workspaces/settings#execution-mode) or [Agent execution mode](/cloud-docs/agents#configuring-workspaces-to-use-the-agent) for Terraform runs
+
+## Enable Drift Detection
+
+To enable drift detection within a workspace:
+1. Verify that the workspace satisfies the [requirements](#requirements).
+1. Go to the workspace and click **Settings > General**.
+1. Select **Enable** under **Drift Detection**.
+1. Click **Save settings**.
+
+If there are no active Terraform runs in the workspace, Terraform Cloud starts a drift detection assessment right after you enable the feature. If you enable drift detection during a speculative plan, Terraform Cloud will start drift detection soon after that plan's completion. If you enable drift detection during other types of runs, Terraform Cloud will start the first assessment in about 24 hours.
+
+Drift detection never interrupts or interferes with runs. Terraform Cloud triggers a drift detection assessment if at least 24 hours have passed since the last assessment and there are no active runs in the workspace. 
+
+You can start a new run during a drift detection assessment, but it will cancel the current assessment. Terraform Cloud will run the next assessment in 24 hours.
+
+Drift detection runs in the background and does not affect your concurrency limit. Terraform Cloud also monitors and controls drift detection concurrency, so it will not cause issues for large-scale deployments with thousands of workspaces. However, drift detection for organizations with a large number of workspaces may take longer to complete.
```

Comment:
```suggestion
Drift detection runs in the background and does not affect your concurrency limit. Terraform Cloud also monitors and controls drift detection concurrency, so it will not cause issues for large-scale deployments with thousands of workspaces. However, Terraform Cloud performs drift detection assessments in batches, so drift detection may take longer to complete when you enable it in a large number of workspaces.
```